migration_sql.go

```diff
@@ -144,7 +149,11 @@ func runSQLMigration(db *sql.DB, scriptFile string, v int64, direction bool) err
 	}
 	defer f.Close()
 
-	statements, useTx := getSQLStatements(f, direction)
+	statements, useTx, err := getSQLStatements(f, direction)
+
```

Comment:
nit: unnecessary line

Reply:
sorry, the whitespace line is unnecessary

Reply:
Ah, yes will remove